osmaxx/excerptexport/templates/excerptexport/partials/export_detail_string.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n capture_as %}{{ export.get_file_format_display }} /
-{% firstof export.finished_at|date:"Y-m-d H:i:s" export.updated_at|date:"Y-m-d H:i:s" as finished_date %}
```

Comment:
How do I do that, though? With 0062085dfed062d2b87b09978a2e0f526e3e23f6, the datetimes are displayed as e.g.
> March 21, 2017, 4:15 p.m

Is that because
> [...] predefined formats may vary depending on the current locale.

as noted in the [`date:'...'` filter documentation](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#date)?